modules/local/get_synteny_genomes.nf

```diff
@@ -16,12 +16,12 @@ process GET_SYNTENY_GENOMES {
 
     script:
     """
-    if [ ! -d ${synteny_path}/${assembly_classT}/ ] || [ -z "\$(ls -A ${synteny_path}/${assembly_classT}/)" ]
+    if [ ! -d ${synteny_path}${assembly_classT}/ ] || [ -z "\$(ls -A ${synteny_path}${assembly_classT}/)" ]
```

Comment:
With this change, are you requiring `${synteny_path}` to have a trailing `/` (or `${assembly_classT}` to have a leading `/`) ?
Did you run into issues if there were two consecutive `/` ?

Reply:
Yes, I updated it to assume that the trailing / was being set in the .yaml, to be consistent with other .yaml values.

Reply:
The paths taken in by the pipeline all have a trailing `/`. This change just removes a consecutive `/` that would stop any file beign found.